tests/Query/Functions/Sqlite/JsonArrayTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Scienta\DoctrineJsonFunctions\Tests\Query\Functions\Sqlite;
+
+use Scienta\DoctrineJsonFunctions\Tests\Query\SqliteTestCase;
+use Doctrine\ORM\Query\Expr;
+
+class JsonArrayTest extends SqliteTestCase
+{
+    public function testJsonArray()
+    {
+        $this->assertDqlProducesSql(
+            "SELECT JSON_ARRAY(1, 'abc', NULL, b.id, CURRENT_TIME(), CONCAT('a','my')) from Scienta\DoctrineJsonFunctions\Tests\Entities\Blank b",
+            "SELECT JSON_ARRAY(1, 'abc', NULL, b0_.id, CURRENT_TIME, CONCAT('a', 'my')) AS sclr_0 FROM Blank b0_"
```

Comment:
`CONCAT` does not exist in `SQLite`. The doctrine `CONCAT()` function produces `'a' || 'my'`.
The assertion must be:
`SELECT JSON_ARRAY(1, 'abc', NULL, b0_.id, CURRENT_TIME, 'a' || 'my') AS sclr_0 FROM Blank b0_`